codewars5.py

```diff
@@ -0,0 +1,11 @@
+# https://www.codewars.com/kata/576757b1df89ecf5bd00073b/train/python
+# Although the code renders a nice Christmas tree, it does not pass the codewars test.
+# I might have misunderstood part of the assignment. Guidance would be appreciated.
+
+def tower_builder(n_floors):
+    count = 1
+
+    for i in range (n_floors):
+        print("{:^{max_width}}".format("*" * count, max_width = ((n_floors * 2) - 1)))
+        count += 2
+
```

Comment:
I personally think that this solution is very clever and I would accept it. The most important for me in the beginning is that your logic is correct. 
Here are some takeaways why your solution doesn't pass:
1. What your function essentially does is printing a few lines. To pass the tests you need to return the result at the end of the function, not print them
2. Pay attention at the example solutions to the left side of the exercise - those are lists with strings. 
3. I added literally 3 lines to your code and it passed all the tests, here's what you need to do if you want to achieve the same:
a) Declare an empty list, it will be populated with every loop iteration. You can do it like this: christmas_tree = []
b) Instead of printing the whole expression in the brackets, append it to the list. Do: christmas_tree.append(), where inside the brackets you put the whole print content from line 9
c) At the end of the function, return your list that you declared and populated. It won't be empty anymore, it will have all the stars needed.